### PR TITLE
Restructure awesome books into ES6 code

### PR DESCRIPTION
**Pull request by keebot-png**
## The following actions have taken place

- Implemented ES6 syntax. for example, I have refactored my regular functions to Arrow functions.
- Divided my Javascript code into Modules.
- Used Luxon.js library for date and time.
- Passed the linter errors.

### CODE REVIEW

File: modules/store.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(index) {
+    const books = Store.getBooks();
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```
Comment (awais-amjed) [inline modules/store.js]: Kindly replace these class functions with arrow-style syntax following the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#project-requirements) 🤗

File: modules/user-inter.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  static deleteBook(element) {
+    if (element.classList.contains('delete')) {
+      element.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
```
Comment (awais-amjed) [inline modules/user-inter.js]: Kindly replace these class functions with arrow-style syntax following the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#project-requirements) 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @keebot-png , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @keebot-png👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**